PetAdoption-iOS/PetAdoption-iOS/Controllers/filterViewControlelr.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  filterViewControlelr.swift
+//  PetAdoption-iOS
+//
+//  Created by Amir Fleminger on 4/11/18.
+//  Copyright © 2018 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+
+class FilterViewController: UITableViewController {
+    var delegate: FilterSelectorDelegate?
+    enum RowIndex:Int {
+        case All = 0, Dogs, Cats, Birds, SmallFurry, Horses, BarnYard, Reptiles, Rabbits
```

Comment:
The convention for enums in Swift 4 is that cases should begin with lowercase letters